test/connectivity/modules/NetworkController/docker/linux/arm32v7/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG base_tag=1.0.6.0-linux-arm32v7
+ARG base_tag=1.0.6.1-linux-arm32v7
```

Comment:
What's the reason why {NetworkController, TestAnalyzer, TestResultCoordinator, TwinTesters} are the only one needs this non-root user? (except edgeAgent which already has it).

Reply:
iotedge service needs moduleuser or edgehubuser to start the module. Every base image has it. Module base used by many more for example:
https://github.com/Azure/iotedge/blob/master/edge-modules/SimulatedTemperatureSensor/docker/linux/arm32v7/base/Dockerfile